flow/record/adapter/avro.py

```diff
@@ -156,7 +157,7 @@ def descriptor_to_schema(desc):
     return schema
 
 
-def schema_to_descriptor(schema):
+def schema_to_descriptor(schema: dict) -> record.RecordDescriptor:
```

Comment:
```suggestion
def schema_to_descriptor(schema: Dict) -> record.RecordDescriptor:
```
Otherwise I believe we break the 3.7 Python requirement of `flow.record`.